vocabularies/Org.OData.Aggregation.V1.xml

```diff
@@ -136,7 +136,8 @@
         <Annotation Term="Core.Description" String="This property can be used in the `aggregate` transformation" />
       </Term>
 
-      <Term Name="CustomAggregate" Type="Edm.String" Nullable="false" AppliesTo="EntitySet ComplexType EntityContainer">
+      <Term Name="CustomAggregate" Type="Edm.String" Nullable="false" AppliesTo="EntitySet Collection ComplexType EntityContainer">

```

Comment:
```suggestion
      <Term Name="CustomAggregate" Type="Edm.String" Nullable="false" AppliesTo="Collection ComplexType EntityContainer">
```

`Collection` includes `EntitySet`. Do you want to have it redundantly for compatibility reasons?

Reply:
We decided to keep both yesterday. Let's discuss next week.